app/javascript/components/table/TableHead.vue

```diff
@@ -1,32 +1,70 @@
 <template>
   <div class="table-head">
-    <div v-for="(heading, index) in headings"
+    <div 
+      v-for="(heading, index) in headings"
       :key="getVForKey('row', index)"
       class="table-head__cell"
     >
-      {{ heading.title }}
+      <span class="table-head__title-wrapper">
+        <span class="table-head__title">{{ heading.title }}</span>
 
-      <table-sort :sort-key="heading.id" />
+        <tooltip 
+          v-if="!isFirstHeading(index)"
+          :on-hover="false" 
+          :text="getTooltipText(heading.id)"
+          :class="['carousel__tooltip', { 'tooltip--end': isLastHeading(index) }]"
+        >
+          <i class="icon--info-circle block"></i>
+        </tooltip>
+      </span>
+
+      <table-sort 
+        v-if="index !== 0"
+        :sort-key="heading.id" 
+      />
     </div>    
   </div>
 </template>
 
 <script>
 import mixinId from '../../mixins/mixin-ids'
 import TableSort from './TableSort'
+import Tooltip from '../tooltip/Tooltip'
 
 export default {
   name: 'TableHead',
 
-  components: { TableSort },
+  components: { TableSort, Tooltip },
 
   mixins: [ mixinId ],
 
   props: {
     headings: {
-      type: Array, // [ { title: String } ]
+      type: Array, // [ { id: String, title: String } ]
+      required: true
+    },
+    tooltipArray: {
+      type: Array, // [ { id: String, title: String, text: String } ]
       required: true
     }
+  },
+
+  methods: {
+    getTooltipText (id) {
+      const tooltip = this.tooltipArray.find(obj => {
```

Comment:
Find doesn't work in IE - you can loop through manually or use a polyfill. I think one is used in ENCORE